File: src/iOSXamarinBlankApp/iOSXamarinBlankApp/ViewController.cs
using System;

using UIKit;

namespace iOSXamarinBlankApp
{
	public partial class ViewController : UIViewController
	{
		public ViewController (IntPtr handle) : base (handle)
		{
		}

		public override void ViewDidAppear (bool animated)
		{
			var elapsedTimeSinceLaunch = Application.LaunchStopwatch.ElapsedMilliseconds;
			UIAlertView alert = new UIAlertView ("Launch Time", elapsedTimeSinceLaunch + " miliseconds", null, "OK", null);
			alert.Show ();

			base.ViewDidAppear (animated);
		}
	}
}

File: src/iOSNativeBlankApp/iOSNativeBlankApp/ViewController.m
//
//  ViewController.m
//  iOSNativeBlankApp
//
//  Created by Pedro da Rocha Pires on 09/03/16.
//  Copyright © 2016 Pedro da Rocha Pires. All rights reserved.
//

#import "ViewController.h"

extern NSTimeInterval startDate;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

-(void)viewDidAppear:(BOOL)animated{
    NSTimeInterval endDate = [[NSDate date] timeIntervalSince1970];
    NSTimeInterval duration = endDate - startDate;
    
    NSString *message = [NSString stringWithFormat:@"%f miliseconds", duration];
    
    UIAlertView *alert = [[UIAlertView alloc] initWithTitle:@"Elapsed Time" message:message delegate:nil cancelButtonTitle:@"OK" otherButtonTitles:nil];
    [alert show];
}

@end
